index.js

```diff
@@ -12,7 +12,7 @@ function defaultRedactionRules() {
 }
 
 function timestampFunction() {
-  return `,"time":${Math.round(Date.now() / 1000.0)}`
+  return `,"time":${Math.round(Date.now())}`
```

Comment:
In this case we may also remove the Math.round(), since Date.now() returns an "integer"

Reply:
of course! didn't realise